portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -72,7 +75,8 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
               (String) entity.getProperty("content"), 
               (long) entity.getKey().getId(),
               (long) entity.getProperty("likeCount"),
-              (String) entity.getProperty("email"))
+              (String) entity.getProperty("email"),
+              (float) entity.getProperty("sentimentScore"))
```

Comment:
This is a problem that I've recently run into, where ```entity.getProperty("sentimentScore")``` returns an Object, which can't be cast to a primitive float type. Currently my solution is to convert it to a String, but maybe you have a different solution.

Reply:
Nvm, just noticed you brought this up in a different PR!